### ISSUE TITLE
some config parameters of downloaded pre-trained model are different from the config in 'Readme' 

### ISSUE THREAD

**Issue by yanyk13** [ymcui/Chinese-XLNet#40]
> the config of downloaded model('chinese_xlnet_base_pytorch')

Model config {
  "attn_type": "bi",
  "bi_data": false,
  "clamp_len": -1,
  "d_head": 64,
  "d_inner": 3072,
  "d_model": 768,
  "dropout": 0.1,
  "end_n_top": 5,
  "ff_activation": "relu",
  "finetuning_task": null,
  "initializer_range": 0.02,
  "layer_norm_eps": 1e-12,
  **"mem_len": null,**
  "n_head": 12,
  "n_layer": 12,
  "n_token": 32000,
  "num_labels": 2,
  "output_attentions": false,
  "output_hidden_states": false,
  "output_past": true,
  "pruned_heads": {},
  **"reuse_len": null,**
  "same_length": false,
  "start_n_top": 5,
  "summary_activation": "tanh",
  "summary_last_dropout": 0.1,
  "summary_type": "last",
  "summary_use_proj": true,
  "torchscript": false,
  "untie_r": true,
  "use_bfloat16": false
}

> the pre-trained config in 'Readme'

python train.py \
	--record_info_dir=$DATA \
	--model_dir=$MODEL_DIR \
	--train_batch_size=32 \
	--seq_len=512 \
	**--reuse_len=256 \**
	**--mem_len=384 \**
	--perm_size=256 \
	--n_layer=24 \
	--d_model=768 \
	--d_embed=768 \
	--n_head=12 \
	--d_head=64 \
	--d_inner=3072 \
	--untie_r=True \
	--mask_alpha=6 \
	--mask_beta=1 \
	--num_predict=85 \
	--uncased=False \
	--train_steps=2000000 \
	--save_steps=20000 \
	--warmup_steps=20000 \
	--max_save=20 \
	--weight_decay=0.01 \
	--adam_epsilon=1e-6 \
	--learning_rate=1e-4 \
	--dropout=0.1 \
	--dropatt=0.1 \
	--tpu=$TPU_NAME \
	--tpu_zone=$TPU_ZONE \
	--use_tpu=True

My question is why 'mem_len' and 'reuse_len' are null(None) in downloaded models. Thx

**Comment by ymcui** [comment]
Well, it is something with the conversion tools provided by pytorch-bert of huggingface. 
If you would like to do additional pre-training steps, you should change 'null' to the correct numbers. However, if you only use this model for fine-tuning on down-stream tasks, then it is fine to leave as 'null'. 
For example, check 'run_classifier' by XLNet (https://github.com/zihangdai/xlnet/blob/master/run_classifier.py), no `reuse_len` or `men_len` should be provided.

**Comment by yanyk13** [comment]
I see, thanks a lot
